config.yaml

```diff
@@ -32,12 +21,13 @@ adapter_fastp_general: /mnt/ngsnfs/seqres/contaminants/2020_02/adapters/adapters
 adapter_fastp_nextera: /mnt/ngsnfs/seqres/contaminants/2020_02/adapters/NexteraPE-PE.fa
 adapter_fastp_solid: /mnt/ngsnfs/seqres/contaminants/2020_02/adapters/adapters_solid.fa
 adapter_nextera: /mnt/ngsnfs/seqres/contaminants/2020_02/adapters/NexteraPE-PE.fa
-adapter_truseq: /mnt/ngsnfs/tools/miniconda3/envs/wochenende/share/trimmomatic-0.39-2/adapters/TruSeq3-PE.fa
+#adapter_truseq: /mnt/ngsnfs/tools/miniconda3/envs/wochenende/share/trimmomatic-0.39-2/adapters/TruSeq3-PE.fa
+adapter_truseq: /home/hpc/davenpor/programs/miniconda3/envs/wochenende/share/trimmomatic-0.39-2/adapters/TruSeq2-PE.fa
 ea_adapter_fasta: /mnt/ngsnfs/seqres/contaminants/2020_02/adapters/adapters.fa
 # Paths to tools
-path_abra_jar: /mnt/ngsnfs/tools/abra2/abra2_latest.jar
-path_afterqc: /mnt/ngsnfs/tools/afterQC/AfterQC-0.9.6/after.py
-path_alignerboost: /mnt/ngsnfs/tools/dev/Wochenende/dependencies/AlignerBoost.jar
+#path_abra_jar: /mnt/ngsnfs/tools/abra2/abra2_latest.jar
+#path_afterqc: /mnt/ngsnfs/tools/afterQC/AfterQC-0.9.6/after.py
+#path_alignerboost: /mnt/ngsnfs/tools/dev/Wochenende/dependencies/AlignerBoost.jar
```

Comment:
Just a question: why isn't this needed anymore? Is this set somewhere else?

Reply:
No, just not using these tools at present. Abra2.jar has a security issue - log4j - and still hasn't been fixed 6 months later ...